### ISSUE TITLE
Place imports after licence/comment headers

### ISSUE THREAD

**Issue by DenisVuyka** [soates/Auto-Import#25]
For the moment top comments (license headers) are not respected during auto-import. For example:

**sample.ts**
```ts
/*!
 * @license
 * Copyright 2016 Company.
*/

export class MyClass {}
```

During auto-import the line will be added to the top of the file:

**sample.ts**
```ts
import { Observable } from 'rxjs/Rx';
/*!
 * @license
 * Copyright 2016 Company.
*/

export class MyClass {}
```

Expected behaviour: it should put imports after top comment block (ideally to be configurable via some settings flag)

**Comment by soates** [comment]
Working on this as we speak :)

cheers!

**Comment by 19majkel94** [comment]
New imports should be placed at the bottom of the imports.
And according to the [Angular Style Guide](https://angular.io/styleguide#!#03-06), imports should be divided into 2 sections, leaving one empty line between third party imports and application imports.

**Comment by soates** [comment]
Yeah, I agree - I am going to build some configuration into this.

**Comment by electrotype** [comment]
+1

Please add them under both the "/*" and "//" types of comment, if they are before any existing import/declaration.

**Comment by SamVerschueren** [comment]
@soates Is there anything we can do to help you with this feature? If you already have something for this, maybe push it in a new branch so someone else can continue to work on it.

**Comment by leocaseiro** [comment]
Perhaps this extension might help https://marketplace.visualstudio.com/items?itemName=peterjuras.sort-imports
